CardParts/src/Resources/AssetManager.swift

```diff
@@ -0,0 +1,185 @@
+            let path = UIBezierPath(arcCenter: origin, radius: radius, startAngle: radian(from: 270), endAngle: radian(from: 180), clockwise: false)
+            path.lineWidth = linewidth
+            return path
+        }
+        
+        private func triangle() -> UIBezierPath {
+            let size = CGSize(width: 24.0, height: 24.0)
+            let sides = 3
+            let radius: CGFloat = min(size.width, size.height) / 2.0
+            let origin = CGPoint(x: size.width / 2.0 , y:  size.height / 2.0)
+            let startAngle = CGFloat(-1 * (360 / sides / 4))
+            let adjustment = startAngle + CGFloat(360 / sides / 2)
+            
+            let points = polygonPoints(sides: sides, radius: radius, origin: origin, adjustment: adjustment)
+            
+            let path = UIBezierPath()
+            path.move(to: points[0])
+            points.enumerated().forEach { _, point in
+                print(point)
+                path.addLine(to: point)
+            }
+            return path
+        }
+        
+        private func diamond() -> UIBezierPath {
+            let path = UIBezierPath()
+            path.move(to: CGPoint(x: 11.96, y: 3.15))
+            path.addLine(to: CGPoint(x: 11.85, y: 3))
+            path.addLine(to: CGPoint(x: 5.24, y: 12.19))
+            path.addLine(to: CGPoint(x: 5.17, y: 12.29))
+            path.addLine(to: CGPoint(x: 12.24, y: 21.07))
+            path.addLine(to: CGPoint(x: 12.27, y: 21.11))
+            path.addLine(to: CGPoint(x: 18.88, y: 11.81))
+            path.addLine(to: CGPoint(x: 18.95, y: 11.71))
+            path.addLine(to: CGPoint(x: 11.96, y: 3.15))
+            path.close()
+            return path
+        }
+        
+        private func pencil() -> UIBezierPath {
+            let path = UIBezierPath()
+            path.move(to: CGPoint(x: 3, y: 17.25))
+            path.addLine(to: CGPoint(x: 3, y: 21))
+            path.addLine(to: CGPoint(x: 6.75, y: 21))
+            path.addLine(to: CGPoint(x: 17.81, y: 9.94))
+            path.addLine(to: CGPoint(x: 14.06, y: 6.19))
+            path.addLine(to: CGPoint(x: 3, y: 17.25))
+            path.close()
+            path.move(to: CGPoint(x: 20.71, y: 7.04))
+            path.addCurve(to: CGPoint(x: 20.71, y: 5.63),
+                          controlPoint1: CGPoint(x: 21.1, y: 6.65),
+                          controlPoint2: CGPoint(x: 21.1, y: 6.02))
+            path.addLine(to: CGPoint(x: 18.37, y: 3.29))
+            path.addCurve(to: CGPoint(x: 16.96, y: 3.29),
+                          controlPoint1: CGPoint(x: 17.98, y: 2.9),
+                          controlPoint2: CGPoint(x: 17.35, y: 2.9))
+            path.addLine(to: CGPoint(x: 15.13, y: 5.12))
+            path.addLine(to: CGPoint(x: 18.88, y: 8.87))
+            path.addLine(to: CGPoint(x: 20.71, y: 7.04))
+            path.close()
+            return path
+        }
+        
```

Comment:
This is actually getting drawn based of the SVG file for [edit](https://material.io/resources/icons/?search=edit&icon=edit&style=baseline)